src/main/scala/devices/xilinx/xilinxarty100tmig/XilinxArty100TMIG.scala

```diff
@@ -152,7 +152,7 @@ class XilinxArty100TMIG(c : XilinxArty100TMIGParams, crossing: ClockCrossingType
   val depth = ranges.head.size
 
   val buffer  = LazyModule(new TLBuffer)
-  val toaxi4  = LazyModule(new TLToAXI4(adapterName = Some("mem"), stripBits = 1))
+  val toaxi4  = LazyModule(new TLToAXI4(adapterName = Some("mem"), stripBits = 0))
```

Comment:
Default is 0. Please just remove the use of the option, because I'd like to remove the option from toaxi4 completely.

Reply:
Ok I'll fix that